File: src/main/java/net/slipp/repository/smalltalk/SmallTalkRepository.java
package net.slipp.repository.smalltalk;

import java.util.List;

import net.slipp.domain.smalltalk.SmallTalk;
import net.slipp.domain.user.SocialUser;

import org.springframework.data.repository.PagingAndSortingRepository;

public interface SmallTalkRepository extends PagingAndSortingRepository<SmallTalk, Long>{
	List<SmallTalk> findByWriter(SocialUser writer);
}

File: src/main/scala/net/slipp/repository/smalltalk/SmallTalkRepository.scala
package net.slipp.repository.smalltalk

import java.lang.Long
import java.util.List

import net.slipp.domain.user.SocialUser
import net.slipp.domain.smalltalk.SmallTalk
import org.springframework.data.repository.PagingAndSortingRepository

trait SmallTalkRepository extends PagingAndSortingRepository[SmallTalk, Long] {
  def findByWriter(writer: SocialUser): List[SmallTalk]
}